src/applications/edu-benefits/0994/containers/IntroductionPage.jsx

```diff
@@ -4,11 +4,13 @@ import OMBInfo from '@department-of-veterans-affairs/formation-react/OMBInfo';
 import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
 
 import SaveInProgressIntro from '../../../../platform/forms/save-in-progress/SaveInProgressIntro';
+import CallToActionWidget from '../../../../platform/site-wide/cta-widget';
```

Comment:
This can be `platform/site-wide/cta-widget` - recently added some path resolution to make `platform` easier to use.